README.md

```diff
@@ -74,9 +74,23 @@ Add additional notes about how to deploy this on a live system
 Please read [CONTRIBUTING.md](https://gist.github.com/PurpleBooth/b24679402957c63ec426) for details on our code of conduct, and the process for submitting pull requests to us.
 
 
-## Functionality for deleting users whose accounts are older than 100 days
+## Functionality for deleting users whose accounts are older than the desired number of days (100 by default)
 
 The command to run in the terminal, so as to launch the old accounts deleting is :
     php bin/console app:delete-old-accounts
 
-CAUTION : So as to be fully operational, it requires to set a cron on the server.
+CAUTION : So as to be fully operational, it requires to use the cron program on the server.
+
+## Functionality for login with social network
+
+Set up your app on :
+https://developers.google.com/
+https://www.linkedin.com/developers/
+https://developers.facebook.com/apps/
+
+Get your app id and secret foreach social network
+and set up your .env file
+
+## Functionality for reset password
+
+Set up your MAILER_DSN in .env
```

Comment:
+ Add instructions to setup database connection (DATABASE_NAME, _PORT, _USER...etc)